pandas/tests/reshape/test_pivot.py

```diff
@@ -2153,3 +2153,20 @@ def test_pivot_index_none(self):
 
         expected.columns.name = "columns"
         tm.assert_frame_equal(result, expected)
+
+    def test_pivot_index_list_values_none(self):
```

Comment:
I might be wrong but I don't think this is testing the OP...?

I'd use the example from the issue. Do the pivot and test the result of the pivot operation, then test that the arguments have not changed

Reply:
Actually definitely use that. This isn't quite the same use case